docs/api.rst

```diff
@@ -5221,6 +5467,54 @@ ChannelFlags
 .. autoclass:: ChannelFlags
     :members:
 
+AutoModKeywordPresets
```

Comment:
`AutoModKeywordPresets` is a flag type, and I'm relatively sure the others should be fine here as well since all this is at the end of the flags section:
![image](https://user-images.githubusercontent.com/8530778/180660719-c4df8b65-609a-46db-9891-8644e405ffbf.png)

Reply:
Noticed flags are already not all together, so it doesn't really matter, unfortunately.